icepyx/core/Earthdata.py

```diff
@@ -32,7 +33,7 @@ def __init__(
         uid,
         email,
         capability_url,
-        pswd=None,
+        pswd=os.environ.get('EARTHDATA_PASSWORD'),
```

Comment:
wouldn't this mandate that the user set up `EARTHDADA_PASSWORD` and take away the option to log in interactively? Perhaps we could add another `try` option as with the .netrc login capability?

Reply:
Great question!  As written, the `get` function returns `None` if the environmental variable does not exist.  So it is effectively the same as the prior version.

Reply:
Awesome - thanks for that explanation!